### PR TITLE
completed html-forms

### PR DESCRIPTION
**Pull request by bernardino-www**
https://user-images.githubusercontent.com/88168364/129792902-ed6e1fcc-932b-4962-a7e5-aba8e71c6d43.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      .language { color: green; }
+      .signUp { color: skyblue; }
+      .comments { color: blue; }
+      .submit { color: yellow; }
+    </style>
+  </head>
+  <body>
+    <form>
+      <div class="email">
+        <legend>Email:</legend>
```
Comment (codingcodymiller) [inline html-forms/index.html:19]: This should be the `label` element, not a `legend`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <label for="emailPrompt"><input type="email" name="emailPrompt" id="emailPrompt" required></label>
+      </div>
+      <div class="coding">
+        <legend>Why is coding cool?</legend>
```
Comment (codingcodymiller) [inline html-forms/index.html:23]: This should be the `label` element, not a `legend`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        </label>
+      </div>
+      <div class="language">
+        <legend>Favorite Langauge (Choose one):</legend>
+        <label for="languagePrompt"><input type="radio" name="languagePrompt" id="languagePrompt"> HTML</label>
+        <label for="languagePrompt2"><input type="radio" name="languagePrompt2" id="languagePrompt2"> CSS</label>
+        <label for="languagePrompt3"><input type="radio" name="languagePrompt3" id="languagePrompt3"> JS</label>
+      </div>
+      <div class="signUp">
+        <legend>Would you like to sign up for more information?</legend>
```
Comment (codingcodymiller) [inline html-forms/index.html:40]: This should be the `label` element, not a `legend`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <label for="signUpPrompt"><input type="checkbox" name="signUpPrompt" id="signUpPrompt" required> Sign me up!</label>
+      </div>
+      <div class="comments">
+        <legend>Comments:</legend>
```
Comment (codingcodymiller) [inline html-forms/index.html:44]: This should be the `label` element, not a `legend`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        <label for="commentsPrompt"><textarea name="commentsPrompt" id="commentsPrompt" rows="12" cols="35"></textarea></label>
+      </div>
+      <div class="submit">
+        <label for="submitPrompt"><input type="submit" name="submitPrompt" id="submitPrompt" value="Submit"></label>
```
Comment (codingcodymiller) [inline html-forms/index.html:48]: There is no text labelling the submit button, so no `label` element is necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! A few changes for you:

File: html-forms/index.html (hunk 5 above)
Comment (bernardino-www) [reply html-forms/index.html:48]: Why doesn't the `legend` element suffice compared to the `label` element in this context?

File: html-forms/index.html (hunk 1 above)
Comment (bernardino-www) [reply html-forms/index.html:19]: One of the reasons I opted to using `legend` element is that it produces a new line but will make these changes - thanks

File: html-forms/index.html [hunk 6]
```diff
@@ -37,15 +37,15 @@
         <label for="languagePrompt3"><input type="radio" name="languagePrompt3" id="languagePrompt3"> JS</label>
       </div>
       <div class="signUp">
-        <legend>Would you like to sign up for more information?</legend>
-        <label for="signUpPrompt"><input type="checkbox" name="signUpPrompt" id="signUpPrompt" required> Sign me up!</label>
+        <label for="signUpPrompt">Would you like to sign up for more information?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:40]: Almost, the label should actually be around "Sign me up!", this can just be a general text element such as `p`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 6 above)
Comment (bernardino-www) [reply html-forms/index.html:40]: May you please expand on this? Can't decipher what you mean - seems ambiguous

Comment (codingcodymiller) [reply html-forms/index.html]: @bernardino-www No problem! The element you currently have as a `label` should just be a `p` element, And the text "Sign me up!" should be the `label` for this checkbox. You can change the `div` element you have wrapping the element to a `label` like we showed in Q&A, and that should do the trick.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 6 above)
Comment (bernardino-www) [reply html-forms/index.html]: See new changes based off the most recent commit - thanks

Comment (bernardino-www) [reply html-forms/index.html]: Not certain if this is based off my most recent commit: does my recent `b886b3b` commit work?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**